doc/INSTALL.t2t

```diff
@@ -85,22 +85,19 @@ liable to any duties or guarantees. The editors and publishers do not take
 any responsibility or liability for failures and their consequences. You are 
 always welcome for indicating possible mistakes.
 
-You can download this document as part of the QGIS 'User and 
-Installation Guide' in HTML and PDF format via http://qgis.org. A current 
-version is also available at:
+The current instructions apply to the **master** branch of QGIS. If you wish
```

Comment:
@jef-n is there a way to have **master** replaced automatically by the current branch name (so that comment stays coherent with the releases)?